src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx

```diff
@@ -529,7 +549,17 @@ function createManageGitHubAccountButton(container: Explorer): CommandButtonComp
   return {
     iconSrc: GitHubIcon,
     iconAlt: label,
-    onCommandClick: () => container.openGitHubReposPanel(label),
+    onCommandClick: () =>
+      useSidePanel
+        .getState()
+        .openSidePanel(
+          label,
+          <GitHubReposPanel
+            explorer={container}
+            gitHubClientProp={container.notebookManager.gitHubClient}
+            junoClientProp={undefined}
```

Comment:
Is this correct? Passing `undefined`? If so we should change the type of junoClientProp to be optional

Reply:
done